README.md

```diff
@@ -351,6 +351,7 @@ can have some major pitfalls if you're not careful.  To learn more about how to
 properly create live patches, see the [Patch Author
 Guide](doc/patch-author-guide.md).
 
+
```

Comment:
I don't think these spaces show up in the Markdown rendering, but if we want to be consistent with two blank lines, then can you also update the spaces preceding the Installation and License sections.

Reply:
Will do.